### ISSUE TITLE
能否支持一下win10-ARM？桌面版对ARM处理器不支持。

### ISSUE THREAD

**Issue by amvoidhub** [Dreamacro/clash#671]

**Comment by ac1982** [comment]
https://github.com/golang/go/issues/36439 we could follow this issue for   Golang builder for windows 10 arm64.

**Comment by Dreamacro** [comment]
@ac1982 maybe add Windows arm64 on go 1.16

**Comment by zhzy0077** [comment]
The support has been added. Seems will be shipped as part of 1.17.
Test to build a local GO on master. Looks all good:
![image](https://user-images.githubusercontent.com/8717471/118357572-6695e000-b5ad-11eb-8044-81d4272cd8fe.png)

**Comment by ac1982** [comment]
> The support has been added. Seems will be shipped as part of 1.17.
> Test to build a local GO on master. Looks all good:
> ![image](https://user-images.githubusercontent.com/8717471/118357572-6695e000-b5ad-11eb-8044-81d4272cd8fe.png)

Use go 1.7 beta build clash windows arm64 from source master works fine. 

```
sudo docker run --it -v /www/clash-master:/usr/src/myapp -w /usr/src/myapp -e GOOS=windows -e GOARCH=arm64 golang:1.17beta1-alpine3.13
#go env -w GOPROXY=https://goproxy.cn,direct
#go build -v
#cp $GOPATH/bin/windows_arm64/clash.exe /usr/src/myapp/
```
